lib/games/bingo.dart

```diff
@@ -143,6 +144,8 @@ class BingoState extends State<Bingo> with SingleTickerProviderStateMixin {
     print("second call when we are doing p2p");
     _Bingodata = await fetchPairData(
         widget.gameConfig.gameCategoryId, _maxSize * _maxSize);
+    var data = await ActivityTopicDao().getActivitiesByTopicId("lion");
```

Comment:
I am assuming you did this for testing. But it should not be part of your commit